### DISCUSSION TITLE
IOS/Apple Folder Albums

### DISCUSSION THREAD

**Discussion by R-Nabil** [immich-app/immich#12372] [General]
Hey,

Is there a plan to allow for Folder Albums in Immich ? I'm asking in light of the latest feature which allows to sync Albums from iPhone directly to Immich (which works great).

However, my albums are all organised in Folders that make some sort of sense to me, and I can't figure out how to synchronise this to Immich (I end up with a flat structure which makes it a bit cumbersome to navigate). Is this something possible or planned ?

Thanks

PS: As an example, I have sorted pictures of my kids by Age Group (1month to 24 months, then by year etc). These are all under each kid own Folder.

**Comment by radh21301** [comment]
Can I ask what you mean by folders? And what you mean by albums? What is the difference?

**Reply by R-Nabil** [reply]
> Can I ask what you mean by folders? And what you mean by albums? What is the difference?

Folders contains many albums. Just an organisation feature

**Reply by radh21301** [reply]
Oh damn. Never knew we could do that on apple photos. Just checked, and there it is.

Immich does not have the capability to have sub-albums/child-albums. There could already be a feature request for that.

**Reply by radh21301** [reply]
https://github.com/immich-app/immich/discussions/2073


I suppose the feature you ask can only be done after this one is available.

**Reply by R-Nabil** [reply]
> Oh damn. Never knew we could do that on apple photos. Just checked, and there it is.
> 
> Immich does not have the capability to have sub-albums/child-albums. There could already be a feature request for that.

Yeah. It's actually super convenient. At least for me. I have millions of albums named similarly, and having folder to differentiate them helps to make it quicker to find them back.

**Reply by R-Nabil** [reply]
> https://github.com/immich-app/immich/discussions/2073
> 
> 
> I suppose the feature you ask can only be done after this one is available.

Looks to me like this feature is even one step further compared to what I want. Nested albums would be more complicated I believe than folders. 

But I would be happy with that indeed 😃 

Thanks for pointing it out

**Reply by radh21301** [reply]
Yeah. I agree. Looks like the folders are basically just a collection of albums. And folders cannot have photos. They can only contain albums.

I think that works perfectly for me as well. Don't really need the complications of nested albums.

**Comment by Symbiot78** [comment]
question ... I've setup my immich app to sync all my icloud albums... but the albums are not being created in immich.. everything is just uploaded flat. 

"Album Sync
You can sync or mirror an album from your phone to the Immich server on your account. For example, if you select Recents, Camera and Videos album for backup, the corresponding album with the same name will be created on the server. Once the assets from those albums are uploaded, they will be put into the target albums automatically."

am i misunderstanding the above?  shouldn't the albums be created?
Am I doing something wrong? or missing something ?

**Comment by Curt4U** [comment]
I have an otber question…

is fhere chance to sync this structure?
- folder
  - album1
  - Album2
- folder2
   - album3
   - Album4

immich ios app just getting album1, album2, album3, albuum4 for sync
